core/tests/webdriverio_utils/ExplorationEditorMainTab.js

```diff
@@ -420,6 +420,12 @@ var ExplorationEditorMainTab = function() {
     };
   };
 
+  this.checkInvisibilityOfSelfLoopWarningInResponse = async function() {
```

Comment:
Invisibility is a strange word IMO. Maybe checkSelfLoopWarningIsNotShown?

Reply:
Thanks @nikitaevg, I have made the suggested changes.